app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -0,0 +1,318 @@
+
+### 1. Teaching subject
+
+You must have accepted a job as a teacher of languages or physics in an English state secondary school on a contract lasting at least one academic year.  
+
+a) Teachers of all languages offered in English state secondary schools are eligible to apply for the IRP. The language or languages can be combined with another subject, but must make up at least 50% of teaching time. You can combine teaching English with another language, as long as the non-English language makes up at least 50% of your teaching timetable.  
+
+b) Physics can be taught with another subject, but must make up at least 50% of teaching time. Teachers of general science or combined science  are also eligible to apply for the IRP, as long as they are teaching the physics element of general science or combined science. General science or combined science can be taught with another subject, but must make up at least 50% of teaching time.  
+
+c) 'State schools' are schools funded by the UK government. 'Secondary schools' teach children aged 11 to 16 or 11 to 18. You will not be eligible if you teach in a school for pupils aged 16 to 18 only.  
+
+d) An academic year is 3 full school terms: for example, if your teaching job started in September 2023 (the start of the autumn term), it would need to continue until July 2024 (covering the spring and summer terms). 
+
+[Check the type and age range of the school](https://www.get-information-schools.service.gov.uk/) where you’ll be teaching. You can also find out about the [different types of state school in England](https://www.gov.uk/types-of-school). 
+
+### 2. Type of visa
+
+You must come to England on one of the visas listed here.  
+ 
+For the following visas, you must have applied for your visa from outside the UK:  
+
+* [Skilled worker visa](https://www.gov.uk/skilled-worker-visa) 
+
+* [Youth Mobility Scheme](https://www.gov.uk/youth-mobility)  
+
+* [India Young Professionals Scheme visa](https://www.gov.uk/india-young-professionals-scheme-visa) 
+
+* [Family visa](https://www.gov.uk/uk-family-visa) 
+
+* [UK Ancestry visa](https://www.gov.uk/ancestry-visa)  
+
+* [British National (Overseas) visa](https://www.gov.uk/british-national-overseas-bno-visa) 
+
+* [High Potential Individual visa](https://www.gov.uk/high-potential-individual-visa)  
+
+For the following Ukrainian and Afghan visas, you can apply for your visa from within the UK:  
+
+* [Afghan citizens resettlement scheme](https://www.gov.uk/guidance/afghan-citizens-resettlement-scheme) 
+
+* [Afghan Relocations and Assistance Policy](https://www.gov.uk/government/publications/afghan-relocations-and-assistance-policy/afghan-relocations-and-assistance-policy-information-and-guidance) 
+
+* [Ukraine Family Scheme visa](https://www.gov.uk/guidance/apply-for-a-ukraine-family-scheme-visa) 
+
+* [Ukraine Sponsorship Scheme](https://www.gov.uk/guidance/apply-for-a-visa-under-the-ukraine-sponsorship-scheme) 
+
+### 3. Date of arrival 
```

Comment:
again, probably not good practice to have 2 headings with the same text, and given the info provided is the same for trainees AND teachers, could we not just have one section for both?

Reply:
Don't want to combine sections because the process and eligibility is different for teachers and trainees and I think it'll confuse things to blend them together. But I can relabel heads.